jujucrashdump/addons.yaml

```diff
@@ -1,9 +1,21 @@
+crm-status:
```

Comment:
is this format compatible with the old format?
we have addons in FCE, do we have to change them?
will that make the old versions of FCE incompatible with juju crashdump?

Reply:
The old addon format works on with the new juju crashdump.
The new addon format does not necessarily work with the old juju crashdump. But that will not be a problem since only the addons within juju crashdump are updated so far.